agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/processors/AgentProcessor.java

```diff
@@ -36,20 +36,23 @@ public AgentProcessor(@Nullable IncludeExclude include,
         return exclude;
     }
 
+
```

Comment:
NIT:  unnecessary newlines.  Also, there seem to be inconsistency around how we use newline before closing paranthesis in a method.  As far as can tell, there seem to be no newline for most methods.  Can you pl. fix?